config/database.yml

```diff
@@ -56,9 +56,9 @@ development:
 # re-generated from your development database when you run "rake".
 # Do not set this db to the same as development or production.
 test:
-  <<: *default
-  database: CareerAtlas_test
-
+  adapter: postgresql
+  database: travis_ci_test
```

Comment:
@rposborne Your right they did break my local tests but when I changed it travisCI broke.